<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterInputs</class>
 <widget class="QMainWindow" name="EnterInputs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Master</string>
  </property>
  <property name="windowIcon">
   <iconset resource="share/mainResources.qrc">
    <normaloff>:/icons/images/new/network-config.png</normaloff>:/icons/images/new/network-config.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="mainLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="maFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionSave_project"/>
    <addaction name="actionGeneration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="maActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionAddState"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="maFile"/>
   <addaction name="maActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Панель инструментов</string>
   </property>
   <property name="toolTip">
    <string>Панель инструментов</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_project"/>
   <addaction name="actionSave_project"/>
   <addaction name="separator"/>
   <addaction name="actionGeneration"/>
   <addaction name="separator"/>
   <addaction name="actionAddState"/>
   <addaction name="actionAdd_device"/>
   <addaction name="actionLevel_up"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddState">
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/textedit/images/filenew.png</normaloff>:/textedit/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>Add state</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionGeneration">
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/icons/images/new/finish.png</normaloff>:/icons/images/new/finish.png</iconset>
   </property>
   <property name="text">
    <string>Generation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLevel_up">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/icons/images/new/go-up.png</normaloff>:/icons/images/new/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Level up</string>
   </property>
   <property name="toolTip">
    <string>Level up</string>
   </property>
   <property name="shortcut">
    <string>PgUp, Backspace</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/textedit/images/filesave.png</normaloff>:/textedit/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="toolTip">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/icons/images/new/document-open.png</normaloff>:/icons/images/new/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open project</string>
   </property>
   <property name="toolTip">
    <string>Open project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreate_project">
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/icons/images/new/document-new.png</normaloff>:/icons/images/new/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Create project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAdd_device">
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/icons/dev</normaloff>:/icons/dev</iconset>
   </property>
   <property name="text">
    <string>Add device</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="share/mainResources.qrc">
     <normaloff>:/icons/images/test.png</normaloff>:/icons/images/test.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="share/mainResources.qrc"/>
 </resources>
 <connections/>
</ui>
